package.json

```diff
@@ -10,7 +10,8 @@
   },
   "license": "GPL-3.0",
   "dependencies": {
-    "circomlib": "0.2.2"
+    "circomlib": "0.2.2",
+    "wasmsnark": "0.0.10"
```

Comment:
Why wasmsnark is imported in dependencies?

Reply:
wasmsnark is required for the `buildpkey.js` utility which converts the `proving_key.json` into `proving_key.bin`.  It's used in `compile-circuit.sh`.

Reply:
but then can be moved into the devDependencies, right?
The `dependencies` is what will be installed when an external project imports this repository as a library, while the `devDependencies` is what is installed when working directly on this repository. For this repo the only `dependencies` is `circomlib`, the rest should go to `devDependencies`

Reply:
ah, makes sense!